lib/whitehall/form_builder.rb

```diff
@@ -36,7 +38,7 @@ def errors
     def error_list
       @template.content_tag(:ul, "class" => "errors disc") do
         object.errors.full_messages.each do |msg|
-          @template.concat @template.content_tag(:li, msg)
+          @template.concat @template.content_tag(:li, msg, data: track_analytics_data('form-error', "#{object.class.name.demodulize.underscore.dasherize}-error", msg))
```

Comment:
You can calculate this outside the enumerator.

Reply:
Updated